<launch>

<node name="slam_mapgen" pkg="erasor" type="slam_mapgen" output="screen">
    <rosparam param="/map/voxelsize">0.2</rosparam>		
    <rosparam param="/map/target_rosbag">"05_2350_to_2670_w_interval_2_node.bag"</rosparam>

    <!-- ！！！！save_path 看一下 -->
    <rosparam param="/map/slam_map_save_path">"/mnt/d/dataset/dynamic_self/test"</rosparam>  

    <!-- 数据集名字 -->
    <rosparam param="/map/dataset_name">"machi_slam_full"</rosparam>


    <rosparam param="/map/cloud_bin_dir">"/mnt/d/dataset/dynamic_self/nantapark_paper/velodyne_intensity_gt"</rosparam>

    <!-- SLAM给出的位姿 -->
    <rosparam param="/map/poses_url">"/mnt/d/dataset/dynamic_self/test/machi/park_no_distort_featurori_config1_full_filter/poses/poses_withtime.txt"</rosparam>

    <!-- id-timestamp 对应文件-->
    <rosparam param="/map/corr_url">"/mnt/d/dataset/dynamic_self/nantapark_paper/timestamps.txt"</rosparam>

    <!-- 这里写SLAM给出的动态索引文件夹  -->
    <rosparam param="/map/slam_dynamic_indices_dir">"/mnt/d/dataset/dynamic_self/test/machi/park_no_distort_featurori_config1_full_filter/frontend/dynamic_indices"</rosparam>

    <!-- 时间范围限制  -->
    <rosparam param="/map/start_timestamp">0</rosparam>
    <rosparam param="/map/end_timestamp">1999999999</rosparam>
    
    <rosparam param="/map/viz_interval">1</rosparam>
    <rosparam param="/large_scale/is_large_scale">true</rosparam>
</node>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find erasor)/rviz/kitti_mapgen.rviz"/>

</launch>
